config/application.properties

```diff
@@ -1,6 +1,18 @@
 logging.path = logs/
 
 #####################################################################
+# Begin: Google maps configuration
+#####################################################################
+
+google.geocode.client.id = ${GOOGLE_GEOCODE_CLIENT_ID}
```

Comment:
This is now required in your .env to run locally (and you'll have to add it to your config in heroku as well -- I did this for hmhb, hmhb-ryan, and hmhb-demo already, but I don't have access to hmhb-mckoon).